matrixprofile/matrixProfile.py

```diff
@@ -17,11 +17,62 @@
 
 from .scrimp import scrimp_plus_plus
 
+def is_array_like(a):
+    """
+    Helper function to determine if a value is array like.
+    Parameters
+    ----------
+    a : obj
+        Object to test.
+    Returns
+    -------
+    True or false respectively.
+    """
+    return isinstance(a, (list, tuple, np.array))
```

Comment:
np.array SHOULD BE np.ndarray

`return isinstance(a, tuple([list, tuple, np.ndarray]))`